pkg/internal/controllers/test/ready.go

```diff
@@ -0,0 +1,238 @@
+/*
```

Comment:
Question- wouldn't it be better to name these files `_test.go` so they don't get included in the built binary?

Reply:
They will not get put into the built binary because they are not referenced by the program.

Non-referenced func and code is omitted from built Go programs